src/index.html

```diff
@@ -5,9 +5,117 @@
+          <div class="info__text">
+            <p><strong>Over 50 years experience in love.</strong><br> Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+            <p>The market leader in real estate since 2003. <br>Awards Real Estate Company, 2010, 2011, 2013.<br> More than 200 satisfied customers premium segment.<br> We focus only on new buildings and high-level doskanalno know this area better than other specialists. <br>Working directly with developers - so have more useful information about the profitable deals.</p>
+            <p><strong>We will never be late! Guaranteed secure transaction - 100%.</strong></p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading">Gallery<span class="heading__crown"></span></h2>
+        <div class="gallery">
+          <div class="photos photos_wide_left">
+            <img src="images/photos/1.jpg" alt="photo" class="photos__img photos__img--wide">
+            <img src="images/photos/2.jpg" alt="photo" class="photos__img photos__img--short">
+          </div>
+          <p class="gallery__caption">Marina Palms / <a href="" class="link">North Miami Beach, FL 33162</a></p>
+          <div class="photos">
+            <img src="images/photos/3.jpg" alt="photo" class="photos__img">
+            <img src="images/photos/4.jpg" alt="photo" class="photos__img">
+          </div>
+          <p class="gallery__caption">Edition Residens / <a href="" class="link">Miami Beach, FL 33139</a></p class="gallery__caption">
+          <div class="photos photos_wide_right">
+            <img src="images/photos/5.jpg" alt="photo" class="photos__img photos__img--short">
+            <img src="images/photos/6.jpg" alt="photo" class="photos__img photos__img--wide">
+          </div>
+          <p class="gallery__caption">Faena House / <a href="" class="link">Miami Beach, FL 33140</a></p>
+          <button class="button gallery__button">View All</button>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading">Consultation<span class="heading__crown"></span></h2>
+        <div class="consultation">
+          <div class="consultation__text">
+            <h3 class="consultation__heading">How long have <br>you do yourself a gift?</h3>
+              <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          </div>
+          <form class="form consultation__form">
+            <label><input type="text" name="name" placeholder="Name" class="input form__input"></label>
```

Comment:
Переноси для удобства чтения кода
```suggestion
            <label>
              <input
                type="text" 
                name="name" 
                placeholder="Name" 
                class="input form__input"
              >
            </label>
```